cobc/cobc.c

```diff
@@ -3001,6 +3019,80 @@ remove_trailing_slash (char *data)
 	}
 }
 
+static int
+string_is_dash (const char *s)
+{
+	return (strcmp (s, COB_DASH) == 0);
+}
+
+static void
+add_depend_target (const char *s)
+{
+	if (!cb_depend_target) {
+		cb_depend_target = cobc_strdup (s);
+	} else {
+		/* multiple invocations add to the list */
+		const size_t orig_len	= strlen (cb_depend_target);
+		const size_t new_len	= strlen (s);
+		const size_t buff_len	= orig_len + 1 + new_len + 1;
+		cb_depend_target = cobc_realloc (cb_depend_target, buff_len);
+		memset (cb_depend_target + orig_len, ' ', 1);
+		memcpy (cb_depend_target + orig_len + 1, s, new_len);
+		memset (cb_depend_target + orig_len + 1 + new_len, 0, 1);
+	}
+}
+
+static void
+add_depend_escape_target (const char *s)
+{
+	int i, len, nchars;
+	len = strlen (s);
+	nchars = 0;
+	for (i=0; i<len; i++){
+		char c = s[i];
+		if (c == '$' || c == '#' || c == '*') nchars++;
```

Comment:
Do we cover this in the testcases already?

Reply:
Yup, at line 1227.